browser/main/NoteList/index.js

```diff
@@ -11,6 +11,7 @@ import NoteItem from 'browser/components/NoteItem'
 import NoteItemSimple from 'browser/components/NoteItemSimple'
 import searchFromNotes from 'browser/lib/search'
 import fs from 'fs'
+import path from 'path'
 import { hashHistory } from 'react-router'
 import markdown from 'browser/lib/markdownTextHelper'
```

Comment:
please remove `markdown` and `findNoteTitle`, because there are never used.

Reply:
I often get the request to make changes to code I didn't (have to) touch, which sounds like refactoring to me and in my opinion should be done in a separate PR. What is your stance on that? Do you want me to make changes to code that isn't part of the bug I am fixing with my PR, such as this one? In an earlier PR I did just that, and then I was asked to make those changes in a separate PR, so that's why I am reluctant to do it (I saw those unused imports already).

Reply:
The problem that I commented on was caused by the code you removed with this PR. So I thought it should be settled with this PR. But if you do not want it, I'll fix it with another PR.

Reply:
Sorry, you are right, I should've removed the imports as well. I'll do that in a minute.